src/communication/request_client_server.hpp

```diff
@@ -168,10 +168,13 @@ namespace cubcomm
       void start_thread ();	  // start thread that receives and handles requests
       void stop_thread ();	  // stop the thread
 
+      bool has_registered_handlers () const;
```

Comment:
I think it was used to verify that the handlers are initialized before the servers are started, but I am not sure this new function is very useful now.